contribute/architecture/frontend-data-requests.md

```diff
@@ -29,7 +29,7 @@ Migrating the core data sources to the new `fetch` function [is an ongoing proce
 
 ## Request queue
 
-Depending on how the web browser implements the protocol for HTTP 1.1, it will limit the number of parallel requests, lets call this limit _max_parallel_browser_request_.
+Depending on how the web browser implements the protocol for HTTP 1.1, it will limit the number of parallel requests; let's call this limit _max_parallel_browser_request_.
```

Comment:
```suggestion
Depending on how the web browser implements the protocol for HTTP 1.1, it limits the number of parallel requests; let's call this limit _max_parallel_browser_request_.
```
```suggestion
Depending on how the web browser implements the protocol for HTTP 1.1, it will limit the number of parallel requests; let's call this limit _max_parallel_browser_request_.
```